spinta/components.py

```diff
@@ -516,6 +516,7 @@ class Model(MetaData):
         'params': {'type': 'object'},
         'comments': {},
         'uri': {'type': 'string'},
+        'schema_type': {'type': 'string'},
```

Comment:
There should be no such ting, as `schema_type`, this is only used as metadata to know schema type, when loading, but `load` commands should not receive it.